File: newspring/src/main/java/com/example/entiity/Order.java
package com.example.entiity;

import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;
import org.springframework.lang.Nullable;

import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;
import java.sql.Time;

@Table(name = "orderinfo")
@Data
@ToString
@NoArgsConstructor
public class Order {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)//表示自增
    private Integer order_id;
    private String order_name;
    private String order_time;
    private String order_phone;
    private String order_vaccine;
    private String order_addr;

    private String user_name;
}

File: app/src/main/java/com/example/android/entity/Order.kt
package com.example.android.entity

import com.google.gson.Gson

class Order(
    val order_id: Int? = null,
    val order_name: String,
    val order_time: String,
    val order_phone: String,
    val user_name: String? = null,
    val order_vaccine:String,
    val order_addr:String

)
